### DISCUSSION TITLE
Custom Script + Twitch = 405

### DISCUSSION THREAD

**Discussion by mechaneyes** [mozilla/hubs#3212] [Hubs Cloud Questions] [answered]
Hello,

I've got a custom script in a custom client where I'm trying to pull in the audio stream from a Twitch account. Unfortunately I'm getting this 405 error:

> media-url-utils.js:154 HEAD https://my-aws-internal-cors-proxy-doman.com/https://www.twitch.tv 405

The video from the same Twitch account is successfully streaming into the room, so I'm not sure why it balks on the audio. Can anyone shed some light on this?

Thanks!

... updated ...

So you know what i'm doing, here's my edit in `media-views.js`:

	export var selectedUrl = "remote selected";

	...
	
	async  createVideoTextureAudioSourceEl() {
		const url = this.data.src;
		url.includes("m3u8") ? (selectedUrl = url) : "";

And here's the simplified relevant code in my custom script:

	let  passUrl;
	const getUrl = async data  => {
		import("../media-views").then(selectedUrl => {
			passUrl = selectedUrl;
		});
	};

	...

	hls.loadSource(passUrl);

**Comment by emclaren** [comment, marked as answer]
Hi there! I hope you hear back from someone in our community who may have some further guidance. Unfortunately, we are a small team and don't have the capacity to provide support for custom client modifications; however, we will leave this discussion thread open to see if anyone else has any feedback or suggestions. Please let us know if you are able to make any progress on this, as I'm sure our community would be interested!

Side note - while we may not be able to provide support for custom clients, if you are interested in chatting with our dev team to see if they have any hints or suggestions we have a regularly scheduled meetpup where we hang out in Hubs, share what we are working on, answer questions, and tour different spaces. At least a few members of the dev team attend each week, and you would be more than welcome to join - head to the meetup channel in our discord group (https://discord.gg/wHmY4nd ) for a link to join and more information on the schedule (it is currently Fridays from 1130-1230 PT, but this could always change in the future)

**Comment by mechaneyes** [comment]
thanks!
